client/lib/site/index.js

```diff
@@ -50,11 +50,13 @@ Emitter( Site.prototype );
 
 Site.prototype.attributes = function( attributes ) {
 	attributes = attributes || {};
-
-	for ( var prop in attributes ) {
+	for ( const prop in attributes ) {
 		if ( attributes.hasOwnProperty( prop ) ) {
 			this[ prop ] = attributes[ prop ];
 		}
+		if ( 'updates' === prop ) {
+			this.update = attributes[ prop ];
```

Comment:
Yes, but do we want to support older version of Jetpack? 
I think this makes it easier to do so. I don't think we use updates anywhere.